### PR TITLE
Reworked musicbrainz scrape execution order

### PR DESCRIPTION
**Pull request by TimHenkelmann**
Reworked musicbrainz scrape execution order as described in #81

### CODE REVIEW

**Review by kordianbruck — APPROVED** [review]
seems to be fine by my account

**Review by ShilpaGhanashyamGore — APPROVED** [review]
Scraping is forced in the order Artists ,Releases and then works. They don't happen parallel like before.

File: musicbrainz/getArtists.js [hunk 1]
```diff
@@ -138,7 +138,7 @@ module.exports = {
             }
 
 
-        }, 3000)
+        }, 1200)
```
Comment (pfent) [inline musicbrainz/getArtists.js:141]: That intervall time screams *magic number*

Comment (kordianbruck) [reply musicbrainz/getArtists.js:141]: It is - kind of. The website allows a scrape every second.

Comment (TimHenkelmann) [reply musicbrainz/getArtists.js:141]: Yep, and setting it to 1000 would therefore result in a couple of blocked requests

Comment (pfent) [reply musicbrainz/getArtists.js:141]: Just complaining that *constants* are better than magic inline numbers… 🤷‍

Comment (sacdallago) [reply musicbrainz/getArtists.js:141]: Agree with @pfent :) It's defined in a couple locations so having it on top in `const TTLXX` fashion would't be _that_ bad :)

**Pull request merged**